meta-resin-common/recipes-support/resin-supervisor/files/resin-supervisor.service

```diff
@@ -25,6 +25,7 @@ ExecStart=@BASE_BINDIR@/bash -c 'source @SBINDIR@/resin-vars && \
     -v /proc/net/fib_trie:/mnt/fib_trie \
     -v /var/log/supervisor-log:/var/log \
     -v /:/mnt/root \
+    -e DOCKER_ROOT=/mnt/root/var/lib/docker \
```

Comment:
This is needed by the delta update mechanism, Relevant code can be found here: https://github.com/resin-io/resin-supervisor/blob/master/src/config.coffee#L17